src/System.Data.Common/src/System/Data/Selection.cs

```diff
@@ -439,7 +439,7 @@ private void DeleteRecord(int recordIndex, bool fireEvent)
         /// <summary>
         /// When searching by value for a specific record, the DataRow may require backdating to reflect the appropriate state
         /// otherwise on Delete of a DataRow in the Added state, would result in the <see cref="System.Comparison&lt;DataRow&gt;"/> where the row
-        /// reflection record would be in the Detatched instead of Added state.
+        /// reflection record would be in the Deleted instead of Added state.
```

Comment:
Is this change correct?

Reply:
@stephentoub I think you're right, after searching more in the source in System.Data there is a `DataRowState.Detached` so I think should be **Detached**, not **Deleted** or **Detatched**. I'll correct this.